easysnmp/interface.c

```diff
@@ -2226,9 +2233,9 @@ static PyObject *netsnmp_get(PyObject *self, PyObject *args)
 
     if (error)
```

Comment:
Actually, the `if (error) {...}` block can be deleted, since the two cases both converge to `Py_RETURN_NONE;`, so:
```
if (error)
{
    Py_RETURN_NONE;
}
Py_RETURN_NONE;
```
should just be:
```
Py_RETURN_NONE;
```

Going a step further, it looks like all of the cleanup is properly handling this case, so I guess the use of `error` could also be removed.

This basic observation can be applied elsewhere.